src/poker/EquityEvaluator.java

```diff
@@ -8,16 +10,21 @@ public class EquityEvaluator {
     public final List<Card> hand2;
     Deck deck;
     private List<Card> board;
+    public  Player player1;
+    public Player player2;
 
     public HashMap<String, String> equityMap;
 
 
-    public EquityEvaluator(Deck deck, List<Card> hand1, List<Card> hand2, List<Card> board) {
+    public EquityEvaluator(Deck deck, List<Card> hand1, List<Card> hand2, List<Card> board, Player player1, Player player2) {
         this.deck = deck;
         this.hand = hand1;
         this.hand2 = hand2;
         this.board = board;
```

Comment:
zmienne globalne hand , hand2, board nie sa tutaj potrzebne (to bylo bardziej robione na potrzeby pokazania ci jak dziala ten konstruktor itp)

Reply:
funkcje tej klasy powinny przyjmowac handy tak jak to robi np public List<Double> calculateEquityTest(List<Card> hand1, List<Card> hand2)